AD Review: very minor edits (#11)
lwig-wg/terminology: draft-ietf-iotops-7228bis.md

(An alternative term, when the properties as a network node are not in
focus, is "constrained device".)

> "unconstrained node" cannot be used as an antonym, as engineering is

Reviewer: ```suggestion
"Unconstrained node" cannot be used as an antonym, as engineering is
```

* lack of (or severe constraints on) advanced services such as IP multicast.

> More generally, it is about constrained networks whenever at least

Reviewer 2: ```suggestion
More generally, the term "constrained networks" applies whenever at least
```

  their pins for interfaces to application hardware such as digital
  in/out (the latter often Pulse Width Modulation (PWM)-controllable),
  ADC/DACs (analog-to-digital and digital-to-analog converters), etc.

>   "Systems on a Chip" (SoC) is used where this hardware is specialized for an application.  These devices often implement

Reviewer 2: ```suggestion
  The term "System on a Chip" (SoC) can be used where this hardware is specialized for an application.  These devices often implement
```

kilowatts to microwatts, from essentially unlimited to hundreds of
microjoules.

> Instead of defining classes or clusters, a simple statement, using

Reviewer 2: (The RFC editor will smoothen the sentence...)
